api/valor_api/backend/core/datum.py

```diff
@@ -251,3 +252,38 @@ def get_paginated_datums(
     )
 
     return (content, headers)
+
+
+def delete_datums(
```

Comment:
can you add a funcitonal-test for this?

Reply:
This should be covered by this [test](https://github.com/Striveworks/valor/blob/e2152b120173fb0550074db63796de6cbd39f68d/api/tests/functional-tests/crud/test_create_delete.py#L461-L467).